pkg/server/proto/server.proto

```diff
@@ -86,7 +87,9 @@ service Waypoint {
   // ListProjects returns a list of all the projects. There is no equivalent
   // ListApplications because applications are a part of projects and you
   // can use GetProject to get more information about the project.
-  rpc ListProjects(google.protobuf.Empty) returns (ListProjectsResponse);
+  rpc ListProjects(google.protobuf.Empty) returns (ListProjectsResponse) {
+    option (google.api.http) = { get: "/api/v1/projects" };
```

Comment:
I think we may want to change this to `/waypoint/....` to match patterns in other cloud services

Reply:
Additionally, if we want to use those URLs with a prefix in one (or both place), it may be better to handle this at the router/mux level?

Reply:
yeah, the prefix (be it product name and/or version) will be there, it's a question of if the annotations include it or if the router handles it.